tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -66,14 +67,11 @@ def test_compatible_brokers(self, broker_version):
 
         self.processor.wait()
 
-        num_consumed_mgs = self.consumer.total_consumed()
+        wait_until(lambda: self.consumer.total_consumed() > 0, timeout_sec=30, err_msg="Did expect to read a message but got none within 30 seconds.")
 
         self.consumer.stop()
         self.kafka.stop()
 
-        assert num_consumed_mgs == 1, \
```

Comment:
This test went from checking for exactly one message, to checking at least one message.
If you think that is fine (I dont have the full scope so I dont know) then LGTM! :)